joomla/joomla-cms
# Improved media view by adding position:sticky in .media-toolbar

Pull Request for Issue #36545

### Summary of Changes
Improved media view by adding position:sticky in .media-toolbar and in .media-drive. Also added the z-index so that the elements does not overlap. 

### Testing Instructions

Go to media--> Upload more than 20(or any big number) images. After uploading, scroll down, in case of the long list of media files the search and change folder options are not visible on scrolling down. we need may scroll up down for many action :

- change folder
- search
- change view

See #36545 

### Actual result BEFORE applying this Pull Request

![Screenshot (322)](https://user-images.githubusercontent.com/54266411/149171941-f250812b-a1d5-4cf1-96b7-0ad197890eb3.png)


### Expected result AFTER applying this Pull Request
![Screenshot (323)](https://user-images.githubusercontent.com/54266411/149172464-6c087351-95dc-4334-b29e-185ce53f8280.png)

Search and folder structure visible on scrolling down also so there is no need to scroll up.


### Documentation Changes Required
